test/integration/I18n_test.rb

```diff
@@ -176,4 +176,24 @@ class I18nTest < ActionDispatch::IntegrationTest
         assert true
       end
     end
+    
+    test "should choose i18n-de and move back to i18n-en" do
+      post '/user_sessions', user_session: {
+        username: rusers(:jeff).username,
+        password: 'secret'
+      }
+      follow_redirect!
+      get '/dashboard'
+      
+      # Set the language as Deutsch
+      lang=:de
+      get_via_redirect "/change_locale/"+lang.to_s
+      assert_equal "#{lang}", "#{I18n.locale}"
+      
+      # Move back to English(Default locale)
+      lang=:en
+      get_via_redirect "/change_locale/"+lang.to_s
+      assert_equal "#{lang}", "#{I18n.locale}"
+      assert true
```

Comment:
`assert true` is an example assertion which will never fail, so why don't we remove it here and from line 176 too? Thank you!